### DISCUSSION TITLE
LESSON 1 - Request testnet LINK error / Rinkeby Faucet

### DISCUSSION THREAD

**Discussion by albertozac** [smartcontractkit/full-blockchain-solidity-course-py#1520] [Q&A] [answered]
Hello everyone!
I'm just trying to get my 0.1 testnet eth and I can see the transactions but my wallet still has no eth. What's happening?

My public address:

0x38f7301571C678EFdD1980bd1556FdC0BBceD363

Last transaction:

0xd23fea6bf95874e28c828ca915416cb614c999c37051df987295245e938538c7

**Comment by dmi3ich** [comment, marked as answer]
Are you sure you on Rinkeby testnet instead of Mainnet?

**Reply by albertozac** [reply]
yes

**Reply by albertozac** [reply]
It is a problem with Chrome. I've downloaded Brave browser and I can see my eth. what should I do to see them on chrome?

**Reply by dmi3ich** [reply]
Try to remove additional extensions, sometimes them produce errors in console (press F12 and select console to see possible errors), and block other extensions.

**Reply by cromewar** [reply]
yes seems to be a problem with the browser because I checked the transaction and everything seems to be working fine.

**Reply by BasedAltcoin** [reply]
Rinkeby Testnet is offline for me, anyone else?

**Reply by albertozac** [reply]
thank you guys! It was definitely a problem with extensions.
